cryptol-saw-core/src/CryptolSAWCore/CryptolEnv.hs

```diff
@@ -483,31 +578,61 @@ updateFFITypes m env = env { eFFITypes = eFFITypes' }
             "Cannot find foreign function in term environment: " <> Text.pack (show nm)
         ]
 
+-- | bindCryptolModule - ad hoc function called when `D <-cryptol_load` is seen
+--     on the command line.
+--
+--   FIXME:
+--    - submodules are not handled correctly below.
+--    - the code is duplicating functionality that we have with `importModule`
+--   TODO:
+--    - new design in PR#2569 should replace this function such that the
```

Comment:
#2569 is an issue and not a PR, correct? If so, then calling it "PR#2569" is misleading.